mods/ra/sequences/structures.yaml

```diff
@@ -9,6 +9,9 @@ fcom:
 	bib: bib3
 		Start: 0
```

Comment:
For consistency and cleanup, the ```Start: 0``` should go from this file, too, while you're at it.